lib/components/user/monitored-trip/saved-trip-editor.tsx

```diff
@@ -52,7 +54,13 @@ const SavedTripEditor = (props: Props): JSX.Element => {
     return (
       <>
         <PageTitle title={title} />
-        <BackLink />
+        {/* If creating, back button should return to trip planner. If not, it should return
+        to saved trips, just in case the user accessed this page through their email. */}
+        {isCreating ? (
+          <BackLink />
```

Comment:
If we are doing links, how about we place a `<Link to="/">` for when `isCreating` is true? (The link will preserve query params in the URL)

Reply:
I think you're right, added in [da8783a](https://github.com/opentripplanner/otp-react-redux/pull/1255/commits/da8783aa0e80dd3ab09e6b4200fe23a61ac7d8b0)